test/parallel/test-net-server-keepalive.js

```diff
@@ -7,6 +7,13 @@ const server = net.createServer({
   keepAlive: true,
   keepAliveInitialDelay: 1000
 }, common.mustCall((socket) => {
+  const setKeepAlive = socket._handle.setKeepAlive;
+  socket._handle.setKeepAlive = common.mustCall((enable, initialDelay) => {
```

Comment:
Can you please add an assertion to verify that this is called with `2000` and `3000`?

Reply:
> Can you please add an assertion to verify that this is called with `2000` and `3000`?

Thanks for review ! Done !